lib/authorization.js

```diff
@@ -3,14 +3,22 @@ const errors = require('./errors'), rparams = require('./param'), sender = requi
   constants = require('./constants').get(),
   csauth = constants.header.AUTH,
   fields = {}, params = {};
+// initialize [TODO: overridable] global default
+var unauthenticatedcode = constants.httpcodes.UNAUTHENTICATED;
```

Comment:
This should be a `const` instead of a `var`.

Reply:
Actually wouldn't this just be easier to do:

```js
const HttpStatus = require('http-status-codes');
// and later
const checkLoggedIn = (req, res, next, unauthenticatedResponse = {code:HttpStatus.UNAUTHENTICATED}) => {
```

Reply:
It is `var` instead of `const` since it can be overridden in the init function when an option is provided to override the global default is added.

Should we provide that option in the init or should i still make the requested changes?
Either way, Should I move the HttpStatus to a separate file `http-status-codes`?

Reply:
> It is var instead of const since it can be overridden in the init function

Ah, right. So the logic is something like: when returning unauthenticated error, do local override, else `init()` override, else `401`?

Reply:
Yes

Reply:
Well, we have no `init()`-driven override now, so why bother?

Reply:
Let me add that option then :)